src/inspector/models/inspectorlistmodel.cpp

```diff
@@ -293,6 +293,11 @@ void InspectorListModel::subscribeOnSelectionChanges()
             setElementList(QList(elements.cbegin(), elements.cend()));
         });
 
+        m_notation->notationChanged().onNotify(this, [this]() {
+            auto elements = m_notation->interaction()->selection()->elements();
+            setElementList(QList(elements.cbegin(), elements.cend()));
+        });
```

Comment:
this is quite critical - even if what elements are selected hasn't changed, if properties of said elements have, the inspector needs to update itself